python/SeanZvvHbb13TeVSteps/README.md

```diff
@@ -0,0 +1,71 @@
+##Instructions for ZvvHbb on LXPLUS
+
+*Disclaimer: This package is under active development. The default branch, V20_from_silvio, is guaranteed to run at PSI or Pisa. The version that David uses runs on LXPLUS, but may or may not be synchronized with the default branch. The default branch is being tested on LXPLUS and, pending potential modifications and updates, should be verified to run on LXPLUS by the end of next week.*
```

Comment:
are you using lxbatch in any respect? Silvio prepared the bash scripts like Step0_prep.sh
for his convenience because in Pisa he uses a local cluster, 
but there is a "controlling" script calling the runAll.sh
called ./submitThem.py have you ever looked at it?
it is meant to be used for batch submissions, I also added other featuers there...
https://twiki.cern.ch/twiki/bin/viewauth/CMS/HiggsBBcodeETH#Running_on_the_batch_system
We should probably discuss before things diverge...